recipes/libzmq/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class LibZMQConan(ConanFile):
+    name = "libzmq"
```

Comment:
why libzmq and not zeromq? Is there any idea for zeromq as installer package? The folder name is zeromq btw.

Reply:
My reasoning is that the name of the project is libzma: https://github.com/zeromq/libzmq/

Reply:
fair enough!